Server/JokesOnYou.Web.Api/Profiles/JokesProfile.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AutoMapper;
+using JokesOnYou.Web.Api.DTOs;
+using JokesOnYou.Web.Api.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JokesOnYou.Web.Api.Profiles
+{
+    public class JokesProfile : Profile
+    {
+        public JokesProfile()
+        {
+            CreateMap<Joke, JokeReplyDto>()
+                .ForPath(destination => destination.Author.Id, options =>
```

Comment:
Theres no need to do forpath/config for every property. If we would have to do it for every entity/prop automapper becomes kinda useless. The way automapper works is it compares the property names in both entities, if they match it maps the one to the other.

Reply:
ill try without. i thought i did try it without and didn't work but ill try again!

Reply:
Added 
```
CreateMap<User, JokeAuthorDto>();
```
Works now.